### PR TITLE
Compatibility with alternative mailable syntax 

### LINKED ISSUE #21: Compatibility with alternative mailable syntax
We need to update our examples to have both old and new syntax mentioned,

and making sure it works with both.

https://github.com/laravel/framework/pull/44462

### PR DESCRIPTION
**Pull request by PeterOcansey**
resolves #21

- [x] Update README with new mailable syntax

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.0 || ^8.1",
```
Comment (fosron) [inline composer.json:23]: thats is not the way to do it https://getcomposer.org/doc/articles/composer-platform-dependencies.md

**Review by fosron — CHANGES_REQUESTED** [review]
mentioning 8.x and 9.x is not really a good way to mention it, we should name it as old and new syntax maybe, or just use the terminology that laravel uses

**Comment by PeterOcansey** [comment]
Old heading text
Screenshot 2023-08-28 at 1 45 57 PM

New heading text
Screenshot 2023-08-28 at 1 46 33 PM

**Comment by PeterOcansey** [comment]
This should cut it? cc @fosron

**Comment by fosron** [comment]
@PeterOcansey let me think about it. We might skip the old syntax all together.

**Comment by l-alexandrov** [comment]
Any updates on this?

**Comment by fosron** [comment]
Tested the alternate syntax, works ok.

**Review by fosron — APPROVED** [review]

**Pull request merged**